src/connector/src/sink/encoder/json.rs

```diff
@@ -192,7 +203,15 @@ fn datum_to_json_object(
     custom_json_type: &CustomJsonType,
 ) -> ArrayResult<Value> {
     let scalar_ref = match datum {
-        None => return Ok(Value::Null),
+        None => {
+            if let CustomJsonType::BigQuery = custom_json_type
+                && matches!(field.data_type(), DataType::List(_))
+            {
+                return Ok(Value::Array(vec![]));
```

Comment:
Can you include the BigQuery doc link as a comment here to explain why we need to do this?